codecov.yml

```diff
@@ -1,4 +1,4 @@
 ignore:
   - "third_party"
   - "tests"
-  - "*_test.cc"
+  - "**/*_test.cc"
```

Comment:
Good catch!
For lcov reports, this list is used https://github.com/advancedtelematic/aktualizr/blob/19c6f433c7c9ad217c37b25f551613082a05d228/CMakeLists.txt#L341. It has a different format...

Now, the two coverage figures on this PR are closer but still different:
- 77.6% according to lcov (on Jenkins)
- 75.86% according to codecov

But it should not cause too much worry, not a big problem.